### PR TITLE
WIP: Next

### PR DESCRIPTION
**Pull request by AlexandreBonaventure**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [X] Feature
- [ ] Code style update
- [X] Refactor
- [X] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [X] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] All tests are passing
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: vite.config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const path = require('path');
+
+module.exports = {
+  root: 'demo',
+  outDir: 'demo_dist',
+  alias: {
+    '/@src/': path.resolve(__dirname, 'src'),
```
Comment (posva) [inline vite.config.js:7]: Rather something more standard like

```suggestion
    '@/': path.resolve(__dirname, 'src'),
```

**Comment by AlexandreBonaventure** [comment]
@posva this is now ready to review. e2e tests are passing

**Comment by AlexandreBonaventure** [comment]
Hello @posva just wanna make sure you saw that :) any news on that front ?

**Comment by posva** [comment]
@AlexandreBonaventure Sorry, I haven't been able to yet but I should be able to get to it this week! Can you remove the yarn.lock changes? It will make it easier to update from master

**Comment by AlexandreBonaventure** [comment]
@posva all good now ! Although github diff shows yarn.lock changes (maybe cache) I picked the one from `master`. fyi I'm on vacation so I might be slow to answer.

**Comment by AlexandreBonaventure** [comment]
sounds good!

**Comment by posva** [comment]
I think this is good now! I'm closing this and releasing a v2.0.0!

Thanks a lot @AlexandreBonaventure !

**Comment by posva** [comment]
I'm using it here: https://paths.esm.dev/
Let me know if it works for you as well!

**Comment by AlexandreBonaventure** [comment]
Yes, I've just migrated my app to use it, and works perfectly !

**Comment by posva** [comment]
Nice 🎉

**Pull request closed without merging**